crates/aptos-metrics/src/public_metrics.rs

```diff
@@ -1,5 +1,13 @@
 // Copyright (c) Aptos
 // SPDX-License-Identifier: Apache-2.0
 
-// A list of metrics which will be made public to all the partners
-pub const PUBLIC_METRICS: &[&str] = &["aptos_connections", "revision"];
+// A list of metrics which will be made public
+pub const PUBLIC_METRICS: &[&str] = &[
+    "aptos_connections",
+    "revision",
+    "system_name",
+    "system_kernel_version",
+    "system_os_version",
+    "system_total_memory",
```

Comment:
Silly question, where do these come from: `system_total_memory` and `system_physical_core_count`? Do we need to fetch them in `add_system_info`? Or will they come from somewhere else?
- As far as I can tell, we can get them from the library: `sys.total_memory()` and `sys.processors().len()`.
- Should we also track used memory -- `sys.used_memory()`? It might help to compare usage to total and see if anything interesting happens 😄?

Reply:
Not silly at all! We should definitely have these metrics.

I was previously exporting these metrics in `add_system_info`, but then realized they should probably be actual int gauges rather than bundling them together with the non-numeric json metrics. Was unsure where to massage it into

Reply:
Ok I added it back in as an int gauge. Lmk what you think! Tested from my macbook:

```
$ curl localhost:9102/metrics                           
# HELP system_physical_core_count Physical CPU cores
# TYPE system_physical_core_count gauge
system_physical_core_count 10
# HELP system_total_memory Total system memory
# TYPE system_total_memory gauge
system_total_memory 68719476
```